File: Opdracht 1/Go/opdracht.go
package main

import (
	"fmt"
)

func main() {
	counter := 5
	words := true
	i := 0
	for i < counter {
		if words == true {
        		fmt.Println("Hallo Kruitbosch!")
			words = false
		} else {
			fmt.Println("Doei Kruitbosch!")
			words = true
		}
		i++
	}
}

File: Opdracht 1/Rust/opdracht.rs
fn main() {
    let counter = 5;
    let mut words = true;
    let mut i = 0;

    while i < counter {
        if words == true {
            println!("Hallo Kruitbosch!");
	    words = false;
        } else {
            println!("Doei Kruitbosch!");
            words = true;
        }

        i += 1;
    }
}